Makefile

```
@@ -91,7 +91,7 @@ LD = $(TOOLPREFIX)ld
 OBJCOPY = $(TOOLPREFIX)objcopy
 OBJDUMP = $(TOOLPREFIX)objdump
 
-CFLAGS = -Wall -Werror -O -fno-omit-frame-pointer -ggdb
+CFLAGS = -Wall -Wno-error -O -fno-omit-frame-pointer -ggdb
```

```suggestion
CFLAGS = -Wall -O -fno-omit-frame-pointer -ggdb
```

Just delete `-Werror` shall be fine.